eng/release/Scripts/AcquireBuild.ps1

```diff
@@ -60,7 +60,8 @@ try {
         --bar-uri $MaestroApiEndPoint `
         --password $MaestroToken `
         --separated `
-        --verbose
+        --verbose `
+        --continue-on-error
```

Comment:
This `darc gather-drop` command seems to be particularly error prone in the case that it needs to use SAS tokens to access the drop. It will error out even if it ultimately does collect all of the expected files. This switch helps it continue through its gathering process instead of giving up early. The command still returns a non-zero exit code, even though it appears to have collected all of the files, so I have to put a "continue on error" on the pipeline step as well.